roles/openshift/tasks/firewalld.yml

```diff
@@ -0,0 +1,34 @@
+---
```

Comment:
The main goal was to try to reduce duplication between the separate roles, and eliminate some of the non-openshift specific roles we are including (like base_os, repos). Also, the addition of the sdn configuration adds additional issues as well.

The other goal was to try to push some of the duplicated tasks from the playbooks into the role as well, that way adding and maintaining additional host providers should take less work. I'm hoping it will also make doing multi-provider deployments easier and more consistent as well.

My first stab at this, I created an openshift-common role and then set that role as a dependency of both openshift-master and openshift-node. I wouldn't be against going back to that route. Or even adding wrapper roles around the current openshift role to provide the separate master and node roles with more sane defaults for each.